strangerstudios/paid-memberships-pro
# Removed the What's this text from the CVV field

### All Submissions:






### Changes proposed in this Pull Request:

Removed the "What's this" text and popup that shows next to the CVV field for some gateways

Resolves #2279

### How to test the changes in this Pull Request:

1. Select the Testing Only gatway or any other gateway that uses the default CC fields on the checkout page
2. There should no longer be a 'whats this' link next to the CVV field.

### Other information:




### Changelog entry

Removed the 'Whats this' prompt next to the CVV field
